java-strings/src/test/java/com/baeldung/stringtolist/ConvertStringToListUnitTest.java

```diff
@@ -0,0 +1,94 @@
+package com.baeldung.stringtolist;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+
+
+public class ConvertStringToListUnitTest {
+
+    private final ConvertStringToList stringToListConverter = new ConvertStringToList();
+    private final String countries = "Russia,Germany,England,France,Italy";
+    private final String ranks = "1,2,3,4,5, 6,7";
+    private final String others = ",,,,,";
+    private final List<String> expectedCountriesList = Arrays.asList("Russia","Germany","England","France","Italy");
+    private final List<Integer> expectedRanksList = Arrays.asList(1,2,3,4,5,6,7);
+    private final List<String> expectedOthersList = Arrays.asList("","","","","","");
+    
+    @Test
+    public void givenString_thenGetListOfStringByJava() {
+        List<String> convertedList = stringToListConverter.stringToStringListInJava(countries);
+        assertEquals(expectedCountriesList, convertedList);
+    }
+ 
+    @Test
+    public void givenEmptyStrings_thenGetListOfStringByJava() {
+        List<String> convertedList =stringToListConverter.stringToStringListInJava(others);
+        assertEquals(expectedOthersList, convertedList);
+    }
+    
+    @Test
+    public void givenEmptyStrings_thenGetListOfStringByApache() {
+        List<String> convertedList =stringToListConverter.stringToStringListUsingApacheCommonsLang(others);

```

Comment:
We have a formatter for Eclipse that you can run your code through to fix up little mistakes like this `=` without a space

Reply:
Applied formatter.